modules.d/90multipath/module-setup.sh

```diff
@@ -131,6 +122,19 @@ install() {
         fi
     }
 
+    [[ $hostonly ]] || {
+        for_each_host_dev_and_slaves is_mpath \
+            || [[ -f /etc/multipath.conf ]] || {
+            cat > "${initdir}"/etc/multipath.conf << EOF
+# Minimal multipath.conf generated by dracut
+# Avoid any devices being multipathed by default
+defaults {
+         find_multipaths strict
+}
+EOF
```

Comment:
Since this is an install of the module, could we maybe keep a file `multipath.conf` and install it like in[ e.g.](https://github.com/dracut-ng/dracut-ng/blob/ef0972fe5349bdf6e821cb79a47cf412caf99059/modules.d/06rngd/module-setup.sh#L35)?

Reply:
Feel free to do that. IMO this is simple enough to be kept inline.

Reply:
A separate file does not seem to have many benefits here and can be considered a follow-up PR